format.lua

```diff
@@ -32,59 +32,19 @@ local anchor_mt = {
 	---@param snd Anchor
 	---@return boolean
 	__lt = function(fst, snd)
```

Comment:
Please keep the existing `Anchor` and `Span` comparisons. They're structural comparisons, but those are still useful.